src/main/kotlin/com/healthmetrix/labres/SwaggerConfig.kt

```diff
@@ -53,14 +57,32 @@ data class DocumentationInfo(
     val title: String,
     val version: String,
     val description: String,
-    val contact: DocumentationContact
+    val contact: DocumentationContact,
+    val servers: List<Server>
 ) {
     data class DocumentationContact(
         val name: String,
         val url: String,
         val email: String
     )
 
+    data class Server(
+        val url: String,
+        val description: String,
+        val version: String
+    )
+
+    fun toServers(): List<io.swagger.v3.oas.models.servers.Server> {
```

Comment:
Already imported? Not a blocker though

Reply:
hm, the problem is theres a name conflict. there are 2 `Server`s in this file. ill make one more specific